lib/exercism/tooling_job.rb

```diff
@@ -69,7 +69,7 @@ def locked!
       end
     end
 
-    def executed!(status, output, exception)
+    def executed!(status, output)
```

Comment:
What happens with exceptions now?

Reply:
They're written to s3 in the tooling invoker (as part of the metadata upload). That PR will follow once this is merged.

Reply:
👍